srcpkgs/planner/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'planner'
 pkgname=planner
-version=2.6.9
+version=2.7
 revision=1
 build_style=meson
 build_helper="gir"
-hostmakedepends="gettext pkg-config vala glib-devel"
+hostmakedepends="gettext pkg-config vala glib-devel curl"
```

Comment:
According to https://github.com/alainm23/planner/search?q=curl, it seems like curl should be a runtime dependency.

Reply:
Done, thanks.